egs/swbd/s5c/local/reverberate/IR_stats.m

```diff
@@ -0,0 +1,348 @@
+function [rt,drr,cte,cfs,edt] = IR_stats(x, fs, varargin)
+%function [rt,drr,cte,cfs,edt] = IR_stats(filename, varargin)
```

Comment:
We can't include this code. Ask the users to specify this file location and die if you cant find it.